sdk-workflows/src/main/java/io/dapr/workflows/runtime/DaprWorkflowActivityContext.java

```diff
@@ -14,11 +14,12 @@
 package io.dapr.workflows.runtime;
 
 import com.microsoft.durabletask.TaskActivityContext;
+import io.dapr.workflows.WorkflowActivityContext;
 
 /**
  * Wrapper for Durable Task Framework {@link TaskActivityContext}.
  */
-public class WorkflowActivityContext implements TaskActivityContext {
+class DaprWorkflowActivityContext implements WorkflowActivityContext {
```

Comment:
Can this remain as `WorkflowActivityContext` for parity with dotnet? [See the naming convention here](https://github.com/dapr/dotnet-sdk/blob/cfd4fbee84a1b5024928137dcce3a101dff083e1/src/Dapr.Workflow/WorkflowActivityContext.cs#L22)

I'd also personally prefer to keep the file name as is for parity with dotnet, but also bc we have the filepath including dapr already, I dont think that is needed as a prefix in the file name for either of these files: the `daprworkflowactivity...` && `daprworkflowcontext...`

Reply:
@cicoyle thanks for your feedback, I have renamed the classes to `DefaultWorkflowContext` and `DefaultWorkflowActivityContext`. 

I wanted to provide a little bit of context, both `DefaultWorkflowContext` and `DefaultWorkflowActivityContext` are internal classes that are part of the `io.dapr.workflows.runtime` package. The SDK user will have to work with `WorkflowContext` and `WorkflowActivityContext` interfaces, which are exposed via `io.dapr.workflows` package and these interfaces follow the naming convention that you shared.

Please take a look and let me know your thoughts.